packages/react-email/client/next.config.js

```diff
@@ -0,0 +1,17 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  reactStrictMode: true,
+  swcMinify: true,
+  headers: async () => [
+    {
+      source: '/:path*',
+      headers: [{ key: 'Cache-Control', value: 'no-store' }],
+    },
+  ],
+  output: 'standalone',
+  experimental: {
```

Comment:
I believe we don't need to flag the experimental app dir in the latest Next.js version, probably ok to remove it